src/App.js

```diff
@@ -53,9 +53,11 @@ class App extends Component {
         const tags = {};
 
         for(let project of data) {
-          categories[project.category] = "";
+          let category = project.category.split(',')
+          categories[category[0]] = "";
           for(let tag of project.tags) { tags[tag] = "" };
         }
+        
```

Comment:
style nit: remove this extra line. (we'll get a linter set up to do this kind of nitpicky PR feedback at some point)